src/js/components/SchemaField.js

```diff
@@ -231,6 +233,61 @@ class SchemaField extends Component {
     );
   }
 
+  renderPlacement(errorMessage, autoFocus, props) {
+    const {
+      required,
+      name,
+      schema,
+      formData,
+      onChange,
+      onBlur,
+      onFocus
+    } = props;
+
+    // example formData: [[“@zone”,“GROUP_BY”,“3”],[“@zone”,“MAX_PER”,“2"],[“hostname “,”UNIQUE”]]”
+
+    // questions:
+    // 1. when parsing and reducing the formData, can we exactly re-use the constraints marathon JSON parser/reducer?
```

Comment:
These questions are sort of blocking me and @bstavroulakis from progressing any further on this.